RecoParticleFlow/PFClusterProducer/python/particleFlowClusterECAL_cfi.py

```diff
@@ -1,5 +1,265 @@
+        label = cms.untracked.string('ecalPFClusterCorV2_EE_pfSize3_ptbin3_mean_25ns'),
+        tag = cms.string('ecalPFClusterCorV2_EE_pfSize3_ptbin3_mean_25ns')
+      ),      
+      cms.PSet(
+        record = cms.string('GBRDWrapperRcd'),
+        label = cms.untracked.string('ecalPFClusterCorV2_EB_pfSize1_sigma_25ns'),
+        tag = cms.string('ecalPFClusterCorV2_EB_pfSize1_sigma_25ns')
+      ),
+      cms.PSet(
+        record = cms.string('GBRDWrapperRcd'),
+        label = cms.untracked.string('ecalPFClusterCorV2_EB_pfSize2_sigma_25ns'),
+        tag = cms.string('ecalPFClusterCorV2_EB_pfSize2_sigma_25ns')
+      ),
+      cms.PSet(
+        record = cms.string('GBRDWrapperRcd'),
+        label = cms.untracked.string('ecalPFClusterCorV2_EB_pfSize3_ptbin1_sigma_25ns'),
+        tag = cms.string('ecalPFClusterCorV2_EB_pfSize3_ptbin1_sigma_25ns')
+      ),
+      cms.PSet(
+        record = cms.string('GBRDWrapperRcd'),
+        label = cms.untracked.string('ecalPFClusterCorV2_EB_pfSize3_ptbin2_sigma_25ns'),
+        tag = cms.string('ecalPFClusterCorV2_EB_pfSize3_ptbin2_sigma_25ns')
+      ),
+      cms.PSet(
+        record = cms.string('GBRDWrapperRcd'),
+        label = cms.untracked.string('ecalPFClusterCorV2_EB_pfSize3_ptbin3_sigma_25ns'),
+        tag = cms.string('ecalPFClusterCorV2_EB_pfSize3_ptbin3_sigma_25ns')
+      ),
+      cms.PSet(
+        record = cms.string('GBRDWrapperRcd'),
+        label = cms.untracked.string('ecalPFClusterCorV2_EE_pfSize1_sigma_25ns'),
+        tag = cms.string('ecalPFClusterCorV2_EE_pfSize1_sigma_25ns')
+      ),
+      cms.PSet(
+        record = cms.string('GBRDWrapperRcd'),
+        label = cms.untracked.string('ecalPFClusterCorV2_EE_pfSize2_sigma_25ns'),
+        tag = cms.string('ecalPFClusterCorV2_EE_pfSize2_sigma_25ns')
+      ),
+      cms.PSet(
+        record = cms.string('GBRDWrapperRcd'),
+        label = cms.untracked.string('ecalPFClusterCorV2_EE_pfSize3_ptbin1_sigma_25ns'),
+        tag = cms.string('ecalPFClusterCorV2_EE_pfSize3_ptbin1_sigma_25ns')
+      ),
+      cms.PSet(
+        record = cms.string('GBRDWrapperRcd'),
+        label = cms.untracked.string('ecalPFClusterCorV2_EE_pfSize3_ptbin2_sigma_25ns'),
+        tag = cms.string('ecalPFClusterCorV2_EE_pfSize3_ptbin2_sigma_25ns')
+      ),
+      cms.PSet(
+        record = cms.string('GBRDWrapperRcd'),
+        label = cms.untracked.string('ecalPFClusterCorV2_EE_pfSize3_ptbin3_sigma_25ns'),
+        tag = cms.string('ecalPFClusterCorV2_EE_pfSize3_ptbin3_sigma_25ns')
+      ),      
+    )
+)
+
+#pfcluscorsource.connect = cms.string('frontier://FrontierProd/CMS_COND_PAT_000')
+pfcluscorsource.connect = cms.string("sqlite_file:/afs/cern.ch/user/b/bendavid/work/public/ecalPFClusterCorV2_22042015.db")
+
+pfclusprefer = cms.ESPrefer(
+    'PoolDBESSource',
+    'pfcluscorsource',
+    GBRDWrapperRcd = cms.vstring(
+                                'GBRForestD/ecalPFClusterCorV2_EB_pfSize1_mean_50ns',
```

Comment:
so much for a light training, 20 records to get the data out.
For the future, is there a way for GBRForest to support categories from just one object?